spec/models/login_user_spec.rb

```diff
@@ -5,7 +5,7 @@
     context "user does not have github username, name, or email" do
       it "updates from auth hash" do
         github_id = '1234'
-        user = build(:user, github_login: nil, name: nil, email: nil, github_id: github_id)
+        user = build(:user, github_login: nil, name: "", email: "", github_id: github_id)
```

Comment:
Does our Rubocop style guide prefer '' over "" for strings that don't have interpolation? I don't remember anymore